src/css/global.css

```diff
@@ -11,12 +12,12 @@
 }
 
 body {
-    user-select: none;
     font-family: var(--viz-type-face-base);
     font-weight: var(--viz-type-weight-normal);
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
     font-smoothing: always;
     text-rendering: optimizeLegibility;
     text-decoration-skip-ink: auto;
+    user-select: none;
```

Comment:
Y not tho?

Reply:
Vizia's made to feel like a native app, not a website, you can't highlight in apps. It's an edge case but I've seen a mouse accidentally jogged on the dogfooding screens, which ended up with a gnarly highlight over everything